### PR TITLE
feat: add REST for lookup & embed field for new, edit record

### PR DESCRIPTION
**Pull request by jagannath7775**
1. add REST for lookup
2. embed field for new, edit record

### CODE REVIEW

File: config/apps/event/rest/index.js [hunk 1]
```diff
@@ -15,3 +15,11 @@ export function getData(path) {
 // }
 
 export const getAllUsers = getData.bind(null, 'users')
+
+export function getLookupData(path) {
+  const URL = `https://event.test.bitpod.io/svc/api`
```
Comment (bitpod-io) [inline config/apps/event/rest/index.js:20]: hard-coded url.

File: config/apps/event/content.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ import eventRecurringSession from './gql/eventRecurringSession.gql'
 import eventTasks from './gql/eventTasks.gql'
 import eventRegistrationForm from './gql/eventRegistrationForm.gql'
 import registrationType from './gql/registrationType.gql'
-import { getData } from './rest'
+import { getData, getLookupData } from './rest'
 // import { getBPMNData } from './rest/bpmn.js'
```
Comment (bitpod-io) [inline config/apps/event/content.js:18]: Remove commented code.

File: config/apps/event/content.js [hunk 3]
```diff
@@ -1906,14 +1906,29 @@ export default {
             inlineEdit: true,
             newForm: true,
             editForm: true,
+            // dataSource: {
```
Comment (bitpod-io) [inline config/apps/event/content.js:1909]: Remove commented code.

**Review by bitpod-io — CHANGES_REQUESTED** [review]

File: config/apps/event/rest/index.js (hunk 1 above)
Comment (jagannath7775) [reply config/apps/event/rest/index.js:20]: kapil has implemented this, but his changes are not in master. 
after his changes in master, will change this

File: config/apps/event/content.js (hunk 2 above)
Comment (jagannath7775) [reply config/apps/event/content.js]: resolved

File: config/apps/event/content.js (hunk 3 above)
Comment (jagannath7775) [reply config/apps/event/content.js]: resolved

**Review by neeraj-gulati — APPROVED** [review]

**Pull request merged**